polytope/polytope.py

```diff
@@ -442,6 +468,134 @@ def text(self, txt, ax=None, color='black'):
         _plot_text(self, txt, ax, color)
 
 
+def _translate(polyreg, d):
+    """Translate a polyreg by a vector in place. Does not return a copy.
```

Comment:
Indeed, I was too quick to link to the second page. [This "in place"](https://docs.python.org/3.7/tutorial/datastructures.html) from Python's documentation is used in the sense of "in-place" (on the same instance).